ami-recycle-bin/example-pattern.json

```diff
@@ -0,0 +1,55 @@
+{
+    "title": "Deregister expired AMIs and its snapshot with AWS Lambda and retain them in Amazon EC2 Recycle Bin",
+    "description": "This project demonstrates a pattern to deregister and retain expired AMI and its snapshot with AWS Lambda and Amazon EC2 Recycle Bin",
+    "language": "YAML",
+    "level": "200",
+    "framework": "Terraform",
+    "introBox": {
+      "headline": "How it works",
+      "text": [
+        "In this pattern an Amazon EventBridge triggers AWS Lambda function to deregister AMI, delete the associated EBS snapshot and move them to Recycle Bin for retention. The template creates all the necessary resources including Amazon EventBridge Rule, AWS Lambda function, and Recycle Bin rules for AMI and EBS snapshots retention",
+        "AWS Lambda function automates the expiration of Amazon Machine Images (AMIs) moving the AMIs and its associated snapshot to Recycle Bin. Recycle Bin is a feature in Amazon Elastic Compute Cloud (EC2) that allows you to retain AMIs that you have deregistered for a specified retention period, providing an opportunity to recover them if needed. To recover the deleted AMI, its associated snapshot should be recovered first. The Lambda function also adds corresponding tags to both, the AMI and the EBS Snapshot before moving them to Recycle Bin for recovery."
```

Comment:
Please see comments from README

Reply:
> Please see comments from README

Updated as per the suggested changes same as README